tigrc

```diff
@@ -365,7 +365,7 @@ color "    Acked-by"		yellow	default
 color "    Tested-by"		yellow	default
 color "    Reviewed-by"		yellow	default
 color default			default	default	normal
-color cursor			white	green	bold
+color cursor			color15	green	bold
```

Comment:
I'd prefer to use the ANSI color names in the default configuration. These days I am using white on red, inspired by Mutt if I remember correctly, would that be acceptable?